terraform/testnet/indexer.tf

```diff
@@ -11,10 +11,11 @@ module "indexer" {
 
   oidc_provider = module.validator.oidc_provider
 
-  subnet_ids = module.validator.aws_subnet_private.*.id
+  subnet_ids = var.indexer_db_publicly_accessible ? module.validator.aws_subnet_public.*.id : module.validator.aws_subnet_private.*.id
   vpc_id     = module.validator.vpc_id
 
-  db_password = var.indexer_db_password
+  db_password            = var.indexer_db_password
```

Comment:
Is the public access only protected by password? do we have any IP gating?

Reply:
Just added var that controls the security group: `db_sources_ipv4`